common/h/dyn_regs.h

```diff
@@ -124,1891 +128,2114 @@ namespace Dyninst
+        DEF_REGISTER(wsp,      0 | D_REG  |SPR | Arch_aarch64, "aarch64");
+        DEF_REGISTER(pc,       32| FULL   |SPR | Arch_aarch64, "aarch64");
+        DEF_REGISTER(pstate,   2 | D_REG  |SPR | Arch_aarch64, "aarch64");
+        DEF_REGISTER(xzr,		 3 | FULL   |SPR | Arch_aarch64, "aarch64");
+        DEF_REGISTER(n,   N_FLAG | BIT    |FLAG| Arch_aarch64, "aarch64");
+        DEF_REGISTER(z,   Z_FLAG | BIT    |FLAG| Arch_aarch64, "aarch64");
+        DEF_REGISTER(c,   C_FLAG | BIT    |FLAG| Arch_aarch64, "aarch64");
+        DEF_REGISTER(v,   V_FLAG | BIT    |FLAG| Arch_aarch64, "aarch64");
+        DEF_REGISTER(wzr,		 3 | D_REG  |SPR | Arch_aarch64, "aarch64");
+        DEF_REGISTER(fpcr,     4 | D_REG  |SPR | Arch_aarch64, "aarch64");
+        DEF_REGISTER(fpsr,     5 | D_REG  |SPR | Arch_aarch64, "aarch64");
+
+    }	//end of aarch64 namespace
+    namespace amdgpu_vega{
+        //0xff000000  0x00ff0000      0x0000ff00      0x000000ff
+        //arch        reg cat:GPR     alias&subrange  reg ID
+        const signed int SGPR           = 0x00010000;
+        const signed int SGPR_VEC2      = 0x00020000;
+        const signed int SGPR_VEC4      = 0x00030000;
+        const signed int SGPR_VEC8      = 0x00040000;
+        const signed int SGPR_VEC16     = 0x00050000;
+
+        const signed int VGPR           = 0x00060000;
+        const signed int VGPR_VEC2      = 0x00070000;
+        const signed int VGPR_VEC4      = 0x00080000;
+        const signed int VGPR_VEC8      = 0x00090000;
+        const signed int VGPR_VEC16     = 0x000A0000;
+
+        const signed int HWR            = 0x000B0000;
+        const signed int TTMP_SGPR      = 0x000C0000;
+        const signed int FLAGS          = 0x000D0000;
+        const signed int PC             = 0x000E0000;
+        const signed int SYSREG         = 0x00100000;
+
+        // aliasing for flags
+        // if we found out that it is a flag, we no longer need to use the cat  0x00ff0000
+        // so we use thhat part to encode the low offset in the base register
+        //
+
+
+
+        const signed int BITS_1       = 0x00000100;
+        const signed int BITS_2       = 0x00000200;
+        const signed int BITS_3       = 0x00000300;
+        const signed int BITS_4       = 0x00000400;
+        const signed int BITS_6       = 0x00000500;
+        const signed int BITS_7       = 0x00000600;
+        const signed int BITS_8       = 0x00000700;
+        const signed int BITS_9       = 0x00000800;
+        const signed int BITS_15      = 0x00000900;
+        const signed int BITS_16      = 0x00000A00;
+        const signed int BITS_32      = 0x00000B00;
+        const signed int BITS_48      = 0x00000C00;
+        const signed int BITS_64      = 0x00000D00;
+        const signed int BITS_128     = 0x00000E00;
+        const signed int BITS_256     = 0x00000F00;
+        const signed int BITS_512     = 0x00001000;
+
+#include "amdgpu_vega_sys_regs.h"
     }
 
+    namespace amdgpu_cdna{
```

Comment:
Let's make all of the added constants `constexpr int`.

Reply:
I'm not sure why this is out dated, Maybe the file is edited in the main repo? 
Right now the highlight is on a namespace line so I'm not sure what needs to be constexpr int.

Reply:
It looks like I added the comment to one of the older commits in the branch and the file was subsequently updated, so it labelled it as outdated.

>Right now the highlight is on a namespace line so I'm not sure what needs to be constexpr int.

That comment was to change the declaration of the namespace members. In a previous commit, they were declared `static const signed int`, but now they're just `const signed int`. Making them `static` wasn't useful, so I was recommending to change them to `constexpr`. You still could, but it's not strictly necessary.